src/main/java/de/wi2020sebgroup1/cinema/controller/RegistrationController.java

```diff
@@ -0,0 +1,70 @@
+package de.wi2020sebgroup1.cinema.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import de.wi2020sebgroup1.cinema.entities.User;
+import de.wi2020sebgroup1.cinema.repositories.UserRepository;
+import de.wi2020sebgroup1.cinema.usermanagement.UserServiceInterface;
+import de.wi2020sebgroup1.cinema.usermanagement.UserToBeRegistered;
+
+@RestController
+public class RegistrationController {
+
+    private final UserServiceInterface userService;
+    private final UserRepository userRepository;
+
+    private static final Logger LOG = LoggerFactory.getLogger(RegistrationController.class);
+
+    public RegistrationController(UserServiceInterface userService, UserRepository userRepository) {
+        super();
+        this.userService = userService;
+        this.userRepository = userRepository;
+    }
+
+    @GetMapping("/registration")
+    public ResponseEntity<?> showRegistrationForm() {
+        UserToBeRegistered u = new UserToBeRegistered();
+
+        return ResponseEntity.ok(u);
+    }
+    
+    @GetMapping("/registration/confirm")
+    public ResponseEntity<String> confirmRegistration(@RequestBody User user) {
+
+        userService.saveRegisteredUser(user);
+
+        return ResponseEntity.accepted().body("Welcome, " + user.getFirstName() + "! Your account has been created.");
+    }
+
+    @GetMapping("/login")
```

Comment:
Warum ein GetMapping? Ist Login nicht eher ein secure PUT?

Reply:
Hab das so bei Jonas gesehen - können auch Put machen, aber denke dass es logisch ist, weil wir ja eine Antwort erwarten in irgendeiner Hinsicht

Reply:
Was sind hier die Übergabeparameter? 
Also ich bekomme die Antwort, dass was nicht richtig ist, aber weiß auch nicht was ich übergeben sollte. Laut Signatur kann man auch nichts übergeben